src/overlays/actors/ovl_En_Door/z_en_door.h

```diff
@@ -4,14 +4,57 @@
 #include <ultra64.h>
 #include <global.h>
 
+/**
+ * Actor Parameters
+ * 
+ * |                  |       |             |
+ * | Transition Index | Type  | Double Door | Switch Flag OR Text Id - 0x0200
+ * |------------------|-------|-------------|---------------------------------
+ * | 0 0 0 0 0 0      | 0 0 0 | 0           | 0 0 0 0 0 0 
+ * | 6                | 3     | 1           | 6
+ * |
+ * 
+ * Transition Index     1111110000000000
+ * Type                 0000001110000000
+ * Double Door          0000000001000000
+ * Switch Flag          0000000000111111
+ * Text id - 0x0200     0000000000111111
+ * 
+ */
+
+typedef struct {
+    /* 0x00 */ s16 sceneNum;
+    /* 0x02 */ u8 displaylistIdx;
+    /* 0x04 */ s16 objectId;
+} EnDoorInfo;
+
+typedef enum {
+    /* 0x00 */ DOOR_0,          // loads rooms
```

Comment:
maybe make it clear that 0, 1 and 7 are the same "default" behavior?
one could assume they're just unknown behavior especially with keeping digits

Reply:
I hoped to remove 2 (previously 1, but that was a typo) and 7 and leave gaps in the enum with a comment saying `// unimplemented` or something to that affect but when searching for `ACTOR_EN_DOOR` uses in scenes it appears that at least 2 is used, so I left them in for completeness. It should be clear enough that they refer to the same behavior as they are just numbered versions of the same name.